package.json

```diff
@@ -19,41 +19,41 @@
     "create-react-class": "^15.7.0",
     "css-loader": "^6.8.1",
     "elasticlunr": "^0.9.5",
-    "gatsby": "^5.11.0",
-    "gatsby-plugin-catch-links": "^5.11.0",
-    "gatsby-plugin-google-gtag": "^5.11.0",
-    "gatsby-plugin-image": "^3.11.0",
-    "gatsby-plugin-less": "^7.11.0",
-    "gatsby-plugin-manifest": "^5.11.0",
-    "gatsby-plugin-offline": "^6.11.0",
-    "gatsby-plugin-postcss": "^6.11.0",
-    "gatsby-plugin-react-helmet": "^6.11.0",
-    "gatsby-plugin-sharp": "^5.11.0",
-    "gatsby-plugin-typography": "^5.11.0",
-    "gatsby-remark-autolink-headers": "^6.11.0",
-    "gatsby-remark-copy-linked-files": "^6.11.0",
+    "gatsby": "^5.12.5",
+    "gatsby-plugin-catch-links": "^5.12.0",
+    "gatsby-plugin-google-gtag": "^5.12.0",
+    "gatsby-plugin-image": "^3.12.0",
+    "gatsby-plugin-less": "^7.12.0",
+    "gatsby-plugin-manifest": "^5.12.0",
+    "gatsby-plugin-offline": "^6.12.0",
+    "gatsby-plugin-postcss": "^6.12.0",
+    "gatsby-plugin-react-helmet": "^6.12.0",
+    "gatsby-plugin-sharp": "^5.12.0",
+    "gatsby-plugin-typography": "^5.12.0",
+    "gatsby-remark-autolink-headers": "^6.12.0",
+    "gatsby-remark-copy-linked-files": "^6.12.0",
     "gatsby-remark-embed-youtube": "0.0.7",
-    "gatsby-remark-images": "^7.11.0",
-    "gatsby-remark-prismjs": "^7.11.0",
-    "gatsby-remark-responsive-iframe": "^6.11.1",
-    "gatsby-remark-smartypants": "^6.11.0",
-    "gatsby-source-filesystem": "^5.11.0",
-    "gatsby-transformer-javascript-frontmatter": "^5.11.0",
-    "gatsby-transformer-json": "^5.11.0",
-    "gatsby-transformer-remark": "^6.11.0",
+    "gatsby-remark-images": "^7.12.0",
+    "gatsby-remark-prismjs": "^7.12.0",
+    "gatsby-remark-responsive-iframe": "^6.12.0",
+    "gatsby-remark-smartypants": "^6.12.0",
+    "gatsby-source-filesystem": "^5.12.0",
+    "gatsby-transformer-javascript-frontmatter": "^5.12.0",
+    "gatsby-transformer-json": "^5.12.0",
+    "gatsby-transformer-remark": "^6.12.0",
     "global-modules": "^2.0.0",
     "gray-matter": "^4.0.3",
     "json-loader": "^0.5.7",
     "jsonata": "^2.0.3",
     "jsonfile": "^6.1.0",
-    "less": "~4.1.3",
+    "less": "~4.2.0",
```

Comment:
Could you please another ticket to unlock module versions?
There're 2 lock modules(less, progress) but I think we can unlock them.

History:
https://github.com/enactjs/docs/pull/117
https://github.com/enactjs/docs/pull/25

Reply:
I've reverted the `less` dependency update and created a new ticket to unlock the module version for `less` and `progress`.